src/main/resources/Diagram/DiagramViewSheet.xml

```diff
@@ -319,6 +325,26 @@ require.config({
     diagramViewerDeferred.reject();
   });
 
+  // Return rawDiagram (non-XML) if is any diagram in URL
+  function getDiagramFromURL() {
+    if (window.location.hash !== '') {
+      // We need to exlude first 2 letters from hash (#R).
```

Comment:
What if the first 2 letters are not ``#R``? I suspect that a different encoding might be used in that case so we shouldn't try to decompress (we'll get exceptions most probably).

Reply:
So you suggest to add a check for the first 2 letters?
Also, I'm thinking that a `switch` statement can also be an option, for covering further versions.

Reply:
> So you suggest to add a check for the first 2 letters?

Yes.

> Also, I'm thinking that a switch statement can also be an option, for covering further versions.

I would start with an ``if`` and move to ``switch`` later when we add support for more (a switch with a single case feels a bit strange).